lib/AST/RequirementMachine/RequirementLowering.cpp

```diff
@@ -556,6 +734,7 @@ TypeAliasRequirementsRequest::evaluate(Evaluator &evaluator,
   assert(!proto->hasLazyRequirementSignature());
 
   SmallVector<Requirement, 2> result;
+  SmallVector<RequirementError, 2> errors;
```

Comment:
You can diagnose these here too -- this request runs for protocols defined in source